### ISSUE TITLE
canonical interface 

### ISSUE THREAD

**Issue by itdependsnetworks** [networktocode/netutils#345]
### Environment
* Python version:  3.10
* netutils version:  latest

### Expected Behavior

convert Serial0/0/0:0 into Se0/0/0:0

### Observed Behavior

Serial0/0/0:0 is missed

### Steps to Reproduce
1. abbreviated_interface_name("Serial0/0/0:0")

**Comment by itdependsnetworks** [comment]
I would guess tripped up on colon.

**Comment by qduk** [comment]
This doesn't look to be an issue anymore. 

```python
>>> from netutils.interface import abbreviated_interface_name
>>> abbreviated_interface_name("Serial0/0/0:0")
'Se0/0/0:0'
>>>
```
We also have it in the [abbreviated interface test](https://github.com/networktocode/netutils/blob/3b1004eaa66b00bf85475b3a8254c580e2a097e0/tests/unit/test_interface.py#L167) 


```

**Comment by qduk** [comment]
Checked the other way too just in case.

```python
>>> from netutils.interface import canonical_interface_name
>>> canonical_interface_name("Se0/0/0:1")
'Serial0/0/0:1'
```

**Comment by itdependsnetworks** [comment]
Fixed in #346 for future reference.
